src/Qir/Execution/Tests.Microsoft.Quantum.Qir.Tools/QirDriverGeneratorTests.cs

```diff
@@ -0,0 +1,184 @@
+                    {
+                        Name = "UsePauliArrayArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "PauliArrayArg", Type = DataType.ArrayType, ArrayType = DataType.PauliType}}
+                    }
+                },
+                {
+                    "UseRangeArg",
+                    new EntryPointOperation
+                    {
+                        Name = "UseRangeArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "RangeArg", Type = DataType.RangeType}}
+                    }
+                },
+                {
+                    "UseRangeArrayArg",
+                    new EntryPointOperation
+                    {
+                        Name = "UseRangeArrayArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "RangeArrayArg", Type = DataType.ArrayType, ArrayType = DataType.RangeType}}
+                    }
+                },
+                {
+                    "UseResultArg",
+                    new EntryPointOperation
+                    {
+                        Name = "UseResultArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "ResultArg", Type = DataType.ResultType}}
+                    }
+                },
+                {
+                    "UseResultArrayArg",
+                    new EntryPointOperation
+                    {
+                        Name = "UseResultArrayArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "ResultArrayArg", Type = DataType.ArrayType, ArrayType = DataType.ResultType}}
+                    }
+                },
+                {
+                    "UseStringArg",
+                    new EntryPointOperation
+                    {
+                        Name = "UseStringArg",
+                        Parameters = new List<Parameter>{new Parameter{ Name = "StringArg", Type = DataType.StringType}}
+                    }
+                },
+                {
+                    "UseMiscArgs",
+                    new EntryPointOperation
+                    {
+                        Name = "UseMiscArgs",
+                        Parameters = new List<Parameter>{
+                            new Parameter{ Name = "BoolArg", Type = DataType.BoolType},
+                            new Parameter{ Name = "IntegerArrayArg", Position = 1, Type = DataType.ArrayType, ArrayType = DataType.IntegerType},
+                            new Parameter{ Name = "RangeArg", Position = 2, Type = DataType.RangeType},
+                            new Parameter{ Name = "StringArg", Position = 3, Type = DataType.StringType}
+                        }
+                    }
+                }
+            };
+
+        private static string RemoveLineEndings(string str) =>
+            str.Replace("\r\n", string.Empty).Replace("\n", string.Empty).Replace("\r", string.Empty);
```

Comment:
"str.Replace("\r\n", string.Empty)" is redundant. The other two will take care of this case.

Reply:
Fixed the redundant part.